### DISCUSSION TITLE
Governance structure for Ibis

### DISCUSSION THREAD

**Discussion by wesm** [ibis-project/ibis#3165] [General]
Since we're acquiring more Ibis contributors, and none of the active maintainers are a part of Cloudera where the project originated, we should discuss how to handle project governance going forward. No urgency to arrive at a final decision. 

Some options:

* Considering Ibis part of the pandas "organization" and subject to pandas's governance (https://github.com/pandas-dev/pandas-governance)

* Creating independent governance docs for Ibis

* Submitting Ibis to the Apache Incubator (--> Apache Ibis). 

I consider the last a sort of nuclear option -- if we had more commercial software vendors involved with development, it would be the best option, but while our developer group is small it is probably not needed until the community grows larger.

**Comment by xmnlab** [comment]
is there any update related to `Governance structure for Ibis`?

**Comment by jreback** [comment]
we should create a repo and update the docs.
